utils/cryptography/encryption.go

```diff
@@ -0,0 +1,54 @@
+package crytography
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"errors"
+	"io"
+	"os"
+
+	"golang.org/x/crypto/nacl/secretbox"
+)
+
+// EncryptionKey for encrypting secrets
+var EncryptionKey, _ = base64.StdEncoding.DecodeString(os.Getenv("ENCRYPTION_KEY"))
+
+// Both what the encryption key length should be
+var keyLength = 32
+
+// EncryptMessage uses SecretBox to encrypt the message
```

Comment:
is this only used for encrypting 24-byte merchant keys? fyi https://github.com/golang/crypto/blob/master/nacl/secretbox/secretbox.go#L29-L31 is clear that the max message size should be ~16kb, otherwise this function needs to handle chunking. may be worth throwing an error if the message is too long just in case this gets reused later.

Reply:
Ah! That's a good point, I had no idea that was the case. I added a check in this commit https://github.com/brave-intl/bat-go/pull/347/commits/6396ea948031c9eb0580a66442e3e3342bb731de